clients/python/llmengine/completion.py

```diff
@@ -72,6 +77,32 @@ async def acreate(
                 Whether to return the log probabilities of generated tokens.
                 When True, the response will include a list of tokens and their log probabilities.
 
+            repetition_penalty (Optional[float]):
```

Comment:
I checked, they have the same intention, but different implementation. The range is also different, `repetition_penalty` takes [1, infinity), `frequency_penalty` takes [-2, 2]. I thought it will be confusing if we simply replace the name

Reply:
let me take a look. i think exposing all these parameters would be confusing to end users

Reply:
looks like TGI's penalty uses division like in https://arxiv.org/pdf/1909.05858.pdf and vLLM and lightLLM uses minus. let's remove `repetition_penalty` and provide only `presence_penalty` and `frequency_penalty`

Reply:
TGI only supports `repetition_penalty`. Does this mean if users choose TGI framework, they won't have to option to penalize repetition?

Reply:
yes, i would suggest anyone wants to use repetition penalty to migrate to vLLM.